config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties

```diff
@@ -2172,6 +2172,11 @@
 # Default: true
 # contentreview.expose.errors.to.ui=false
 
+# If true and if the provider requires a EULA, then the EULA must be accpeted before a user can submit
+# to an assignment. Otherwise, if set to false, a user can submit to an assignment without accepting the EULA.
+# Default: false
+# contentreview.submission.eula.required=false
```

Comment:
This should be true

Reply:
why should this be true? I chose false to be the default since it would not block any students submissions if a content review service provides a EULA. ATM, only TurnitinOC uses the EULA feature.

Reply:
If the default is falsem, this should be:
contentreview.submission.eula.required=true

Check the other properties. This value should be the opposite of the default value.

Reply:
oh ok! I thought you wanted me to change the default for this property. I have updated the example to true.

Reply:
Thanks! :)